### PR TITLE
Fix ocp 44765

### PR DESCRIPTION
**Pull request by huiran0826**
Resolve ticket https://issues.redhat.com/browse/OCPQE-10576,
Analysis for original issue please see comments in above ticket. 

Test log:
Cluster without proper workers: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/Runner-v3-smoke/4450/console
General cluster:  https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/Runner-v3-smoke/4452/console

/cc @openshift/team-sdn-qe

### CODE REVIEW

File: features/upgrade/sdn/sctp-upgrade.feature [hunk 1]
```diff
@@ -11,6 +11,7 @@ Feature: SDN sctp compoment upgrade testing
   @proxy @noproxy @disconnected @connected
   Scenario: Check the sctp works well after upgrade - prepare
     Given I switch to cluster admin pseudo user
+    And the cluster has workers for sctp
```
[the remark this answers is not in the document]
Comment (zhaozhanqi) [reply features/upgrade/sdn/sctp-upgrade.feature:14]: ah,  misunderstanding.

**Comment by weliang1** [comment]
@huiran0826 Current SCTP script only load sctp modules in worker nodes and test sctp between worker nodes. For testing a cluster where the nodes are both master and worker, we can add one more step after https://github.com/openshift/verification-tests/blob/master/features/step_definitions/networking.rb#L1246 to load sctp module in master node too. Here is the YAML file to load sctp in master nodes:
`apiVersion: machineconfiguration.openshift.io/v1
kind: MachineConfig
metadata:
  labels:
    machineconfiguration.openshift.io/role: master
  name: load-sctp-module1
spec:
  config:
    ignition:
      version: 2.2.0
    storage:
      files:
        - contents:
            source: data:,
            verification: {}
          filesystem: root
          mode: 420
          path: /etc/modprobe.d/sctp-blacklist.conf
        - contents:
            source: data:text/plain;charset=utf-8,sctp
          filesystem: root
          mode: 420
          path: /etc/modules-load.d/sctp-load.conf`

**Comment by huiran0826** [comment]
@weliang1 , Thanks for the suggestion!  Actually I thought about this by updating sctp yaml to match master nodes.  But from the docs https://docs.openshift.com/container-platform/4.10/networking/using-sctp.html  and existing testing codes, I think the sctp mainly focus on worker nodes, not master nodes. 
BTW, it's not simply to add the step to https://github.com/openshift/verification-tests/blob/master/features/step_definitions/networking.rb#L1246. 
I guess maybe we can update the go sctp case to cover this if need. WDYT?

**Comment by weliang1** [comment]
> @weliang1 , Thanks for the suggestion! Actually I thought about this by updating sctp yaml to match master nodes. But from the docs https://docs.openshift.com/container-platform/4.10/networking/using-sctp.html and existing testing codes, I think the sctp mainly focus on worker nodes, not master nodes. I guess maybe we can update the go sctp case to cover this if need. WDYT?

@huiran0826 That's why existing automation only test workers, but not masters. Agree, we can skip the testing sctp automation in SNO until sctp is required to be supported in SNO in the future.

**Pull request merged**